streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -1103,6 +1103,7 @@ boolean sendingOldValueEnabled() {
         final SubscriptionResolverJoinProcessorSupplier<K, V, VO, VR> resolverProcessorSupplier = new SubscriptionResolverJoinProcessorSupplier<>(
             primaryKeyValueGetter,
             valSerde == null ? null : valSerde.serializer(),
+            valueHashSerdePseudoTopic,
```

Comment:
Could we move the initialization of `valueHashSerdePseudoTopic` closer to where it is firstly used?

Reply:
This thought is reasonable, but I put all the pseudo-topic definitions together so that we could see what they all are at a glance.

Reply:
Plus, the first use is the very first processor in the FK join, so it couldn't move down by much anyway :)